lib/datasource/local/settings_storage.dart

```diff
@@ -208,6 +215,15 @@ class _SettingsStorage {
     final format = NumberFormat.simpleCurrency(locale: Platform.localeName);
     return format.currencyName ?? currencyDefaultCode;
   }
+
+  void _updateFirebaseToken(String oldAccount, String newAccount) {
```

Comment:
The setting storage should have no knowledge of firebase FirebaseMessageTokenRepository().